imports/ui/annotation.coffee

```diff
@@ -1,5 +1,7 @@
-annotateContent = (content, annotations, options={})->
-  { startingIndex, endingIndex, selectedAnnotationId } = options
+export annotateContent = (content, annotations, options={}) ->
+  { startingIndex, endingIndex, selectedAnnotationId, pointerEvents} = options
+  pointerEvents ?= true
```

Comment:
@jgoley The classNames arrays should come from the annotations like the attributes property. The selectedAnnotationId and pointerEvents parameters can be replaced by adding the required classNames before the annotations are passed in. The goal is to make it so the sites where the annotation functions are called from have more control over how the content is annotated.